tornado_prometheus_exporter/__init__.py

```diff
@@ -45,5 +45,6 @@ def log_request(self, handler):
             .labels(
                 handler=type(handler).__name__.lower(),
                 method=handler.request.method.lower(),
+                uri=handler.request.uri.lower(),
```

Comment:
There is a big problem here. The URI can include templated parts like primary keys for REST APIs, for instance `/tariffs/1/`, which would lead to prometheus creating a new time series for each _object_, which is both useless and draining Prometheus' performance if there are tens of thousands of objects.

If one wants a solution for this, one would need to get all the URI templates from the web framework and parse URIs accordingly. I am pretty sure that the java library you are using is doing this and I know that the one we are using for aiohttp web servers also does this.

The way I see this:

*  I do not have enough time to implement this here, also given that Tornado is deprecated for me and I will not put time into that framework anymore.
* You could migrate to aiohttp, which is quite simple, but requires knowledge about both frameworks to be efficient. (took me less than 2 hours for our project)
* Someone else could implement it here.
* We leave out the URI part and stay happy with the handler name, which virtually implements that feature.
* "But we are not using templated URIs at friday" is not an accepted argument :stuck_out_tongue: